sr_robot_launch/launch/sr_ur_arm_hand.launch

```diff
@@ -90,10 +91,10 @@
     <rosparam command="load" file="$(find sr_description)/hand/config/$(arg hand_id)_controller_gazebo.yaml"/>
     <include file="$(find ros_ethercat_model)/launch/joint_state_publisher.launch" />
     <!-- ARM -->
-    <rosparam if="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_trajectory_controller.yaml" command="load"/>
-    <rosparam unless="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_trajectory_controller_no_wrist.yaml" command="load"/>
+    <rosparam if="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_$(arg arm_controller_type)_controller.yaml" command="load"/>
+    <rosparam unless="$(arg include_wrist_in_arm_controller)" file="$(find sr_robot_launch)/config/gazebo/controller/$(arg arm_id)_$(arg arm_controller_type)_controller_no_wrist.yaml" command="load"/>
     <group if="$(arg arm_trajectory)">
```

Comment:
Given that we are inside this `if arm_trajectory` I'm not sure that it makes sense to add an argument below to allow for another type of controller?
To be consistent we might have to change this `if`

Reply:
Fixed in 4529bc1e06372b03226014d39b6eb406063550b7.